client_lib/pulp/client/commands/repo/cudl.py

```diff
@@ -522,16 +522,22 @@ def _default_summary_view(repo_list, prompt):
 
     if isinstance(repo_list, dict) and repo_list != {}:
         id_value = repo_list['id'] + ' '
-        name_value = repo_list['display_name']
```

Comment:
Well I had it like this, but I thought avoiding a bunch of spaces is better. Maybe they don't really matter...